app/test-client/index.html

```diff
@@ -4,14 +4,50 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
-    <title>Test Client</title>
+    <title>Ignacius - World's best streamer 2k19</title>
+    <style>
+      body {
+        width: 100%;
+        height: 100vh;
+        overflow: hidden;
+        color: #ffffff;
+        background-color: rgb(101, 0, 196);
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-items: center;
+        font-family: Arial, Helvetica, sans-serif;
+      }
+      video {
+        margin-top: 15px;
+        width: 75%;
+        height: 50%;
+      }
+    </style>
   </head>
   <body>
-    <h1>Ignacius, the P2P coordinator test client.</h1>
+    <h1>START YER STREAM YOU NACHO.</h1>
+    <button id="share">Share screen</button>
     <script src="https://unpkg.com/peerjs@1.0.0/dist/peerjs.min.js"></script>
+    <script src="https://webrtc.github.io/adapter/adapter-latest.js"></script>
+    <script src="https://cdn.firebase.com/v0/firebase.js"></script>
+    <script src="https://cdn.webrtc-experiment.com/getScreenId.js"></script>
+    <script src="https://cdn.webrtc-experiment.com/screen.js"></script>
     <script>
-      const peer = new Peer({ host: 'localhost', port: 8000 });
-      peer.connect();
+      const peer = new Peer('ignacius', { host: 'localhost', port: 8000 });
```

Comment:
Shouldn't be hardcoded, or should at least be on the default port?

Reply:
`Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://localhost:8000/peerjs/id?ts=15714830210200.6787841462309826. (Reason: CORS request did not succeed).`

Reply:
This is hard-coded for testing purposes. If required, I can change this to be dynamic, but I don't see the point since this won't be available in any other environment except the development one.

Reply:
Should at least document it for dev purposes :)

Reply:
Documented!